eternum-docs/src/components/ResourceTable.tsx

```
@@ -0,0 +1,71 @@
+        <tbody>
+          {resourceTable.map((resource) => {
+            return (
+              <tr key={resource.resource_type}>
+                <td className="border-b">
+                  <ResourceIcon size={100} id={resource.resource?.id || 0} name={resource.resource?.trait || ""} />
+                </td>
+                <td className="text-center border-b">{resource.amount}</td>
+                <td className="py-2 border-b">
+                  <div className="flex flex-col gap-4 justify-between">
+                    {resource.cost.map((cost) => (
+                      <div className="grid grid-cols-2">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)